tools/build/suites/weekly/beta14_list

```diff
@@ -0,0 +1,66 @@
+AIOLIST="webapi-noneservice-tests
```

Comment:
No, please don't do such!

Because this file is for Crosswalk-14 branch only, let's call it as `tools/build/suites/WEEKLY`. See https://github.com/crosswalk-project/crosswalk-test-suite/pull/2515#discussion_r34966691